components/Post.tsx

```diff
@@ -35,8 +37,11 @@ export default function Post({ cmsData }: PostOrPageProps): JSX.Element {
         <meta name="description" content={post.excerpt} />
       </Head>
       <chakra.article maxW="1200px" mx="auto">
-        <Box
-          className="post-header item is-hero is-image"
+        <MotionBox 
+    //      animate={{y:-40}}
```

Comment:
nit: removing commented out code if we don't need it

Reply:
:+1: